tests/framework/helpers/HtmlTest.php

```diff
@@ -1729,6 +1729,34 @@ public function testActiveTextInput_placeholderFillFromModelTabular()
         $this->assertContains('placeholder="Name"', $html);
     }
 
+    public function testOverrideSetActivePlaceholder()
+    {
+        $model = new HtmlTestModel();
+
+        $html = MyHtml::activeTextInput($model, 'name', ['placeholder' => true]);
+
+        $this->assertContains('placeholder="My placeholder: Name"', $html);
+    }
+}
+
+/**
+ * Class MyHtml
+ * @package yiiunit\framework\helpers
+ */
+class MyHtml extends Html{
```

Comment:
Why two classes in single file?

Reply:
Because this class was used only once. If you want I can transfer it to a separate file.

Reply:
It breaks PSR-1. I'm for fixing it (including classes existing before this PR).